.gitignore

```diff
@@ -1,5 +1,4 @@
 *temp*
-assets/
```

Comment:
I see including `assets/` in the repo as a seperate topic from serving testlets directly from the assets package. Not including assets means that the code can't be compiled without pulling down the repo, installing go-bindata, and running the command to generate assets.

Specifically on serving testlets directly. There's no reason that the http handler couldn't check the filesystem first, then assets. You'd get both options. At that point the only advantage of extracting the embedded assets would be to allow an end user to modify them.

[slightly off topic]
Personally, I'm not a fan of the script approach to testlets. The current testlets minimally require bash, and some require other dependencies. All the existing testlets could relatively easily be written in pure Go and they'd be cross platform.

The obvious disadvantage of writing them exclusively in Go means that an end user needs to learn Go to extend the functionality. Perhaps having some builtin, native testlets and the ability to have user supplied testlets would be the best of both worlds.

Of course, that's just my opinion, I am happy to modify this PR in whatever way you think is best.